src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -257,6 +258,37 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         this.fetchFeedItems();
     };
 
+    updateComment = (
+        id: string,
+        text: string,
+        hasMention: boolean,
+        permissions: BoxItemPermission,
+        onSuccess: ?Function,
+        onError: ?Function,
+    ): void => {
+        const { file, api, onCommentUpdate = noop } = this.props;
```

Comment:
Default prop value should live in `defaultProps` so they're consistent across the component.

Reply:
OK, updating this whole file to use that pattern